src/ophys_etl/transforms/decrosstalk_wrapper.py

```diff
@@ -62,7 +62,9 @@ def run(self):
                 plane_pair.append(p0)
 
                 for k in ('roi', 'neuropil'):
+                    out_fname = p0['output_%s_trace_file' % k]
                     if k not in traces_0:
+                        print('skipping %s' % out_fname)
```

Comment:
`ArgSchemaParser` class include a logger, preferable to prints. Example:
https://github.com/AllenInstitute/ophys_etl_pipelines/blob/94d8610013dae5f1a4e67249b5654bd64c13308b/src/ophys_etl/transforms/postprocess_rois.py#L99-L103

Reply:
Don't worry. These are all getting `revert`ed once I finish debugging my latest chicanery.

Reply:
gotcha